### PR TITLE
Pull Request for Issue1610: Migrate jenkins server to internal server.

### PR DESCRIPTION
**Pull request by dretrex**
This is the pull request for changing our jenkins build server.

### CODE REVIEW

**Comment by dretrex** [comment]
This is pretty much ready to go as the other stuff relies on others and should be straight forward to implement.
Assigned to @iainworkman.

File: compositeCommon/AMMobility.pri [hunk 1]
```diff
@@ -24,15 +24,18 @@ linux-g++-64 {
 	}
 }
 
-# Special build paths and options for running on the Jenkins auto-build server (currently at http://beamteam.usask.ca:8080)
+# Special build paths and options for running on the Jenkins auto-build server (currently at http://srv-aci-01.clsi.ca)
 CONFIG(jenkins_build) {
-		# Disable Qt Mobility Video until the Jenkins-machine supports that
-		CONFIG -= mobility
+	# Disable Qt Mobility Video until the Jenkins-machine supports that
+	message("Disabling Qt Mobility.")
```
Comment (iainworkman) [inline compositeCommon/AMMobility.pri]: Are these messages used for debugging, or they a more permanent logging mechanism for the Jenkins server?
(ie. are they still needed?)

Comment (dretrex) [inline compositeCommon/AMMobility.pri]: Yeah, they are there for clear console output on the jenkins server.  I left them in on purpose.

**Comment by iainworkman** [comment]
Looks good to me.
Time Estimate: 5 Minutes

**Pull request merged**